.idea/libraries/R_User_Library.xml

```diff
@@ -0,0 +1,6 @@
+<component name="libraryTable">
```

Comment:
Please remove all the `.idea/` IDE settings

Reply:
done